src/ucp/wireup/wireup_cm.c

```diff
@@ -664,15 +867,38 @@ static unsigned ucp_cm_server_conn_request_progress(void *arg)
     return 1;
 }
 
+static ucp_rsc_index_t ucp_listener_get_cm_index(uct_listener_h listener,
+                                                 ucp_worker_h worker)
+{
+    const ucp_rsc_index_t num_cms = ucp_worker_num_cm_cmpts(worker);
+    char cm_name[UCT_TL_NAME_MAX];
+    ucp_rsc_index_t i;
+
+    for (i = 0; i < num_cms; i++) {
+        if (worker->cms[i].cm == listener->cm) {
+            ucs_debug("server received a connection request on the %s "
+                      "sockaddr transport (worker=%p cm=%p worker_cms_index=%d)",
+                      ucp_context_get_cm_name_by_index(worker->context, i,
+                                                       cm_name, UCT_TL_NAME_MAX),
+                      worker, listener->cm, i);
```

Comment:
pls move the debug message to relevant place, this func just matches indexes

Reply:
it finds the cm on which the request was received and prints a message when found.
but ok.. move to outside this function after it returns?

Reply:
> move to outside this function after it returns?

yes, where conn request is actually handled